daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/svalue/OrderingSpec.scala

```diff
@@ -198,9 +201,9 @@ class OrderingSpec extends WordSpec with Matchers with TableDrivenPropertyChecks
     } yield SRecord(Record2TypeCon, record2Fields, ArrayList(x, y))
 
   private def mkVariant(as: List[SValue], bs: List[SValue]) =
-    as.map(SVariant(VariantTypeCon, VariantCon1, _)) ++
-      as.map(SVariant(VariantTypeCon, VariantCon2, _)) ++
-      bs.map(SVariant(VariantTypeCon, VariantCon3, _))
+    as.map(SVariant(VariantTypeCon, VariantCon1, 1, _)) ++
```

Comment:
Why does this constructor rank not start with `0`?

Reply:
Good catch.